timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -291,18 +291,23 @@ module.exports = [
         href: "compression",
         children: [
           {
-            href: "compression-basics"
+            title: "Compression",
+            href: "compression"
           },
           {
+            title: "Manually compress chunks",
             href: "manually-compress-chunks"
           },
           {
+            title: "Decompress chunks",
             href: "decompress-chunks"
           },
           {
+            title: "Backfill historical data",
             href: "backfill-historical-data"
           },
           {
+            title: "Modify a schema",
```

Comment:
Added the titles in to the index, because I noticed they weren't rendering properly automatically:

![image](https://user-images.githubusercontent.com/3914967/120428496-172a1f00-c3b7-11eb-8e09-d6a5c10a833e.png)

Reply:
that is great! this conversion never works in all places, I think keeping it explicit is a good option.